src/LHScriptX/Lexer.h

```diff
@@ -103,8 +104,8 @@ class Token
 	[[nodiscard]] bool IsOP(Operator op) const { return this->type_ == Type::Operator && this->u_.op == op; }
 
 	// todo: assert check the type for each of these?
-	[[nodiscard]] const std::string& Identifier() const { return *this->u_.identifierValue; }
-	[[nodiscard]] const std::string& StringValue() const { return *this->u_.stringValue; }
+	[[nodiscard]] const std::string_view StringValue() const { return this->s_; }
```

Comment:
Returning string_view doesn't seem right, they're just pointers being flung around.

Reply:
Reverted back to string.